src/templates/static/get_started/common.jsx

```diff
@@ -11,7 +11,7 @@ export const Section = ({ id, header, children }) => (
 export const HeaderSecondary = ({header}) => (
     <React.Fragment>
         <SeparatorLine invisible show_mobile className='gr-padding-10' />
-        <h3 className='secondary-color'>{header}</h3>
+        <h3 data-anchor={header} className='secondary-color'>{header}</h3>
```

Comment:
Please do not add it to all `h3` tags, need to be added to selected tags. (using a boolean?)